include/modules/sv_detection_methods/analyze_cigar_method.hpp

```diff
@@ -29,11 +29,11 @@
  *          For more information see the
  *          ([Map Format Specification](https://github.com/samtools/hts-specs/blob/master/SAMv1.pdf)) page 8.
  */
-void analyze_cigar(const std::string & read_name,
-                   const std::string chromosome,
-                   const int32_t query_start_pos,
+void analyze_cigar(std::string const & read_name,
+                   std::string const chromosome,
+                   int32_t query_start_pos,
```

Comment:
- can we pass `chromosome` by reference?
- was it intentional to drop the `const` of `query_start_pos`?

Reply:
No, that was a mistake 😅